src-java/kilda-model/src/test/java/org/openkilda/model/bitops/cookie/CookieSchemaTest.java

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2020 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.model.bitops.cookie;
+
+import org.openkilda.model.Cookie;
+import org.openkilda.model.bitops.BitField;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CookieSchemaTest extends GenericCookieSchemaTest {
+    @Test
+    public void ensureNoEmptyMasks() {
+        try {
+            new BitField(0);
+            Assert.fail();
+        } catch (IllegalArgumentException e) {
+            // expected behaviour
+        }
+    }
```

Comment:
you can use `@Test(expected = IllegalArgumentException.class)` for such tests

```
    @Test(expected = IllegalArgumentException.class)
    public void ensureNoEmptyMasks() {
             new BitField(0);
    }
```

Reply:
ok